src/sql/src/rbac.rs

```diff
@@ -977,6 +978,38 @@ fn generate_rbac_requirements(
                 ..Default::default()
             }
         }
+        Plan::AlterSchemaSwap(plan::AlterSchemaSwapPlan {
```

Comment:
Should AlterClusterSwap be somewhere here too?

Reply:
That was actually introduced in a previous PR, the one where I first added the `ALTER ... SWAP ...` syntax. Here's [the check](https://github.com/ParkMyCar/materialize/blob/main/src/sql/src/rbac.rs#L924-L932) and it enforces the current role must own both clusters, just like we do for schemas.